eslint/rules/jquery-no-trim.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Disallow the use of the deprecated `trim` function',
+      category: 'Possible Errors',
```

Comment:
Can we use custom categorization?
```suggestion
      category: 'jQuery deprecated functions',
```

Reply:
I was actually thinking of a more customised name for `category`.
But, not sure about it since I found this: https://eslint.org/docs/rules/

Reply:
discussed offline. proceeding to update with proposed changes